python/pysmurf/client/base/base_class.py

```diff
@@ -79,12 +79,16 @@ def __init__(self, log=None, epics_root=None, offline=False,
             self.log('Offline mode')
 
 
-        # Setting paths for easier commands - Is there a better way to do this
-        # than just hardcoding paths? This needs to be cleaned up somehow
+        # Setting paths for easier commands - Is there a better way to
```

Comment:
We've been discussing reading the tree back from rogue and populating this. I hate this hard code.